app/src/main/java/org/mozilla/fenix/library/bookmarks/addfolder/AddBookmarkFolderFragment.kt

```diff
@@ -77,8 +77,9 @@ class AddBookmarkFolderFragment : Fragment() {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.confirm_add_folder_button -> {
-                if (bookmark_add_folder_title_edit.text.isNullOrEmpty()) {
-                    bookmark_add_folder_title_edit.error = getString(R.string.bookmark_empty_title_error)
+                if (bookmark_add_folder_title_edit.text?.trim().isNullOrEmpty()) {
```

Comment:
It seems .isNullOrBlank() does basically the same thing?

Reply:
Oh yeah let's use that one sec